src/scripts/browserDeviceProfile.js

```diff
@@ -660,8 +666,12 @@ export default function (options) {
     }
 
     if (canPlayVp9) {
-        if (!browser.iOS) {
+        if (!browser.iOS && (!browser.firefox || !browser.osx)) {
```

Comment:
imo it is easier to read in this form

```suggestion
        if (!browser.iOS && !(browser.firefox && browser.osx)) {
```